client/rest/src/main/java/org/opensearch/client/RequestLogger.java

```diff
@@ -67,11 +68,10 @@ private RequestLogger() {}
      * Logs a request that yielded a response
      */
     static void logResponse(Log logger, HttpUriRequest request, HttpHost host, ClassicHttpResponse httpResponse) {
-        if (logger.isDebugEnabled()) {
-            logger.debug(
-                "request [" + request.getMethod() + " " + host + getUri(request) + "] returned [" + new StatusLine(httpResponse) + "]"
-            );
-        }
+        logger.debug("request [{} {} {}] returned [{}]", () -> request.getMethod(), () -> host,
```

Comment:
```suggestion
        logger.debug("request [{} {}{}] returned [{}]", () -> request.getMethod(), () -> host,
```
In original there's no space between host and URI